packages/test/src/integration-tests/test-update.ts

```diff
@@ -0,0 +1,145 @@
+import * as wf from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers';
+import * as workflows from './workflows';
+
+const test = makeTestFunction({
+  workflowsPath: workflows.workflowsPath,
```

Comment:
I would much rather the workflow definitions reside next to the test code.

Reply:
> I would much rather the workflow definitions reside next to the test code.

Right, good call, I moved them out so that I could share workflows with the interceptor tests (which have to be in separate files). But you're right, I was being lazy and it's better to partially duplicate the workflows and keep test files self-contained.